extensions/amp-story-page-attachment/0.1/amp-story-page-attachment.js

```diff
@@ -250,7 +250,7 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
 
     // Ensures the content of the attachment won't be rendered/loaded until we
     // actually need it.
-    toggle(dev().assertElement(this.containerEl), true);
+    toggle(dev().assertElement(this.containerEl), false);
```

Comment:
It might be better to hide the contents first and then append the stuff, so that reappending things doesn't cause unnecessary layout changes for the page attachment.